Makefile

```diff
@@ -222,9 +223,7 @@ verify-manifests: manifests
 
 verify: verify-codegen verify-mockgen verify-manifests
 
-# before running release, bump the version in OLM_VERSION and push to master,
-# then tag those builds in quay with the version in OLM_VERSION
-release: ver=v$(shell cat OLM_VERSION)
+release: ver=$(GIT_TAG)
 release: manifests
 	@echo "Generating the $(ver) release"
 	docker pull $(IMAGE_REPO):$(ver)
```

Comment:
Is there an ordering issue here? Will this image exist without a tag in the remote repo? If we push a tag before we generate the release manifests, then they won't be present in the release.

Reply:
We should be fine as we always override this `ver` variable during the release process, so the ordering problem shouldn't be an issue as we trigger the release process when a new git tag is created.

Side note: It felt like this target needs to be reworked but I didn't want to tackle any structuring changes in this PR.